### ISSUE TITLE
Build out Wikipedia page

### ISSUE THREAD

**Issue by dajbelshaw** [mozilla/webliteracymap#17]
I've create a Wikipedia 'stub' for Web Literacy: https://en.wikipedia.org/wiki/Web_literacy

Please help me build it out!

**Comment by jamiea** [comment]
That's cool!
              
American Sign Language for 'connect'http://www.lifeprint.com/asl101/gifs-animated/connectto.gif

Date: Mon, 2 Feb 2015 14:45:12 -0800

**Comment by dajbelshaw** [comment]
Would be awesome if someone (you?) could help with these issues flagged on Wikipedia:

![screen shot 2015-02-10 at 13 11 49](https://cloud.githubusercontent.com/assets/2429426/6127753/79ce9668-b126-11e4-991a-5745acf4ba45.png)

![screen shot 2015-02-10 at 13 11 59](https://cloud.githubusercontent.com/assets/2429426/6127755/7eed2b1e-b126-11e4-8be4-1d0da3ad1736.png)

https://en.wikipedia.org/wiki/Web_literacy

**Comment by jamiea** [comment]
Hi Doug,

I haven't contributed to a WP entry before but after a first glance, would it be easier to foster the orphan if the Map competencies (with descriptive explanation) were on the page too? e.g. the existing article  'Web design'  then could link to the appropriate (Design) section of the Map article (This appears to fit in with the WP style guide).

**Comment by dajbelshaw** [comment]
Good point! Just been working on that now, actually. I'll add in those competencies - thanks!

**Comment by jamiea** [comment]
Nice quick work!
Thought I'd mention it just in case, though pretty sure you're on it with the competency descriptions

If I've read the WP style guide correctly I can make a valid link from the article 'Remix' to

'Remixing'
* Modifying existing web resources to create something new

but not to simply
'Remixing'
(Realise that might necessitate restructuring the strand sections)

**Comment by dajbelshaw** [comment]
Cool, so stuff added today:

* Short history of web literacy in the 1990s (so it's not all Mozilla stuff) 
* List of competencies *(per @jamiea's suggestion)*
* Brief history of v1.0 and v1.1
* Lots of references

Please do jump in an edit so it's not all just me!

**Comment by jamiea** [comment]
If it's still needed I'll chip in to getting rid of the orphan tag. Has to be next week though as it's bound to suck me in.

**Comment by dajbelshaw** [comment]
The Wikipedia vortex!

![Vortex](http://media.giphy.com/media/rf5HswhuAeGJy/giphy.gif)

**Comment by jamiea** [comment]
Yeah, had an idle thought about adding some adjectives & became well informed about communities of interest/ innovation !

**Comment by jamiea** [comment]
Looks like the issues flagged earlier have been addressed.

**Comment by PriscillaAlcalde** [comment]
I'd love to contribute somehow, but I confess I'm kind of lost. I tried to connect to the call today, unsuccessfully... btw, I like the wikipedia vortex.

**Comment by jamiea** [comment]
Hi @PriscillaAlcalde 

@dajbelshaw is off this week otherwise I'm pretty sure he would've responded to your post. Also, you can mail him at Mozilla with queries about the Map and/ or contributing to the Web Literacy community.

I think they may have had some technical hitches which might explain why you couldn't connect to the call this week. Do try again next week!

**Comment by dajbelshaw** [comment]
Hi @PriscillaAlcalde! I'm back now. :)

Sorry that you had problems connecting to the community call last week. We'd really appreciate your input! Perhaps you could do one of the following:

- Dive into the [Wikipedia page](https://en.wikipedia.org/wiki/Web_literacy) and start editing (even just the [Talk page](https://en.wikipedia.org/wiki/Talk:Web_literacy))
- Review the skills for v1.5 in [this epic spreadsheet](http://goo.gl/R1tjj3)
